submissions/onyinyeChukwuka/HTML/index.html

```diff
@@ -16,6 +17,7 @@
                     <li> <a href="#"> WOMEN</a></li>
                     <li> <a href="#"> MEN  </a></li>
                     <li> <a href="#"> Hot Sales </a></li>
+                    <li> <a href="../html-forms/contactform.html" target="_blank"> Contact Us </a></li>
```

Comment:
@Onyinye101 Are you sure this works? Your html form folder name is different from what is here